### PR TITLE
feature: added login styles

### PR DESCRIPTION
**Pull request by Tkach7**

### CODE REVIEW

File: src/graphql/client.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { GraphQLClient } from 'graphql-request';
 import { getTokenFromStorage } from '../utils/storage/index';
 
-const endpoint = `${process.env.REACT_APP_API_HOST}/graphql`;
+const endpoint = `http://212.109.192.71:5000/graphql`;
```
[the remark this answers is not in the document]
Comment (Tkach7) [reply src/graphql/client.ts]: i forgot to remove it

File: src/scss/_mixins.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+@mixin mobile-portrait {
+	@media only screen
+	and (min-device-width: 320px)
+	and (-webkit-min-device-pixel-ratio: 2)
+	and (orientation: portrait) {
+		@content
+	}
+}
+
+@mixin mobile-landscape {
```
[the remark this answers is not in the document]
Comment (Tkach7) [reply src/scss/_mixins.scss]: when player loves gaming with landscape ?)

File: src/containers/Login/index.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
+@import '../../scss/mixins';
+@import '../../scss/vars/decorations';
+@import '../../scss/vars/sizes';
```
[the remark this answers is not in the document]
Comment (Tkach7) [reply src/containers/Login/index.scss]: done

File: src/scss/_mixins.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (klikkn) [reply src/scss/_mixins.scss]: ok, but I don't see any specific cases)
will investigate in future

[the remark this answers is not in the document]
Comment (Tkach7) [reply src/scss/_mixins.scss]: ok

**Pull request merged**